Lib/test/test_socketserver.py

```diff
@@ -78,10 +78,19 @@ def setUp(self):
         self.port_seed = 0
         self.test_files = []
 
+        # The ProcessingTCPServer and ProcessingUDPServer both modify the
+        # environment, specifically the "multiprocessing.process._dangling"
+        # WeakSet() of created processes.
+        self._multiproc_dangling = multiprocessing.process._dangling.copy()
```

Comment:
Thinking about this some more, I think you're right.  I've refactored the logic a little bit so that the tests do not need to clean up dangling processes.